sources/packages/web/src/views/institution/InstitutionProfile.vue

```diff
@@ -12,42 +12,340 @@
+            <div class="p-field p-col-6">
+              <label for="primaryAddressPostalCode">Postal/ZIP code</label>
+              <ValidatedInput property-name="primaryAddress.postalCode">
+                <Field
+                  name="primaryAddress.postalCode"
+                  label="Postal/ZIP code"
+                  rules="required"
+                  as="InputText"
+                />
+              </ValidatedInput>
+            </div>
+            <div class="p-field p-col-6">
+              <label for="primaryAddressProvinceState">Province/State</label>
+              <ValidatedInput property-name="primaryAddress.provinceState">
+                <Field
+                  name="primaryAddress.provinceState"
+                  label="Province/State"
+                  rules="required"
+                  as="InputText"
+                />
+              </ValidatedInput>
+            </div>
+            <div class="p-field p-col-6">
+              <label for="primaryAddressCountry">Country</label>
+              <ValidatedInput property-name="primaryAddress.coutry">
+                <Field
+                  name="primaryAddress.coutry"
+                  label="Country"
+                  rules="required"
+                  as="InputText"
+                />
+              </ValidatedInput>
+            </div>
+          </div>
+        </ContentGroup>
+        <HorizontalSeparator />
+        <h4>Privacy Disclaimer</h4>
+        <p>
+          Be continuing into the StudentAidBC Application you agree to
+          StudentAid BC’s Privacy policy.
+        </p>
+        <HorizontalSeparator />
+        <Button
+          type="submit"
+          label="Next"
+          icon="pi pi-save"
+          class="p-mt-4"
+          :disabled="isSubmitting"
+        ></Button>
       </form>
     </template>
   </Card>
 </template>
 
 <script lang="ts">
 import { onMounted, ref } from "vue";
+import { useRouter } from "vue-router";
 import { UserService } from "../../services/UserService";
+import { useForm, Field, useField } from "vee-validate";
+import { useToast } from "primevue/usetoast";
+import * as yup from "yup";
+import { InstitutionProfileState } from "../../types";
+import { InstitutionService } from "../../services/InstitutionService";
+import { InstitutionRoutesConst } from "../../constants/routes/RouteConstants";
+import ValidatedInput from "../../components/generic/ValidatedInput.vue";
+import HorizontalSeparator from "../../components/generic/HorizontalSeparator.vue";
+import ContentGroup from "../../components/generic/ContentGroup.vue";
+
+// These options are temporarily hardcoded here as per team decision.
+// These should be moved to or shared between API/web layer.
+const regulatingBodyOptions = [
+  { name: "Private Act of BC Legislature", value: "private-act" },
+  { name: "ICBC", value: "icbc" },
+  { name: "DQAB", value: "dqab" },
+  { name: "PTIB", value: "ptib" },
+  { name: "ITA", value: "ita" },
+];
 
```

Comment:
regulating body options can also come from a service else like the institution names

Reply:
Institution names are going to be retrieved from a dynamic list from the API.
As per the team decision, the "Regulating body" is going to be static and the API is not even validating it for now, which means that future changes will be introduced.